.gitignore

```diff
@@ -31,6 +31,7 @@ _deps
 # Python build artifacts:
 __pycache__
 *.pyc
+*.pyd
```

Comment:
not sure why this didn't come up before, but while working on this I accidentally pushed a .pyd file which wasn't on the ignore list. better safe than sorry (again), so I added this here